test/test_train_network.py

```diff
@@ -144,7 +142,7 @@ def test_train_model_runs_successfully_for_simplified_case(self):
             "report.txt"
             ]
 
-        existingFiles = os.listdir(self.modelPath)
+        existingFiles = [f.name for f in self.modelPath.iterdir()]
```

Comment:
Just curious, is there any benefit to rewriting this line the new way? The old way I wrote it (`os.listdir(DIR)`) is easier to read (imo); just wondering if there's a specific reason why we should avoid this funciton?

Reply:
@maf88 You marked this as resolved without answering my question. If you want to keep it this way that's fine, I was just trying to understand this a bit deeper.